projects/core/uoft_core/other.py

```diff
@@ -269,74 +269,6 @@ def model_to_yaml(model: "BaseModel"):
     return doc
 
```

Comment:
Will this have any affect on uoft_switchconfig.__main__,py, "from uoft_core.other import Prompt"?

Reply:
Nevermind, excellent :)